fe/fe-core/src/main/java/com/starrocks/sql/plan/PlanFragmentBuilder.java

```diff
@@ -2786,5 +2786,54 @@ public PlanFragment visitPhysicalStreamAgg(OptExpression optExpr, ExecPlan conte
             inputFragment.setPlanRoot(aggNode);
             return inputFragment;
         }
+
+        // TODO: distinguish various stream scan node, only binlog scan now
+        @Override
+        public PlanFragment visitPhysicalStreamScan(OptExpression optExpr, ExecPlan context) {
+            PhysicalStreamScanOperator node = (PhysicalStreamScanOperator) optExpr.getOp();
+            OlapTable scanTable = (OlapTable) node.getTable();
+            context.getDescTbl().addReferencedTable(scanTable);
+
+            TupleDescriptor tupleDescriptor = context.getDescTbl().createTupleDescriptor();
+            tupleDescriptor.setTable(scanTable);
+
+            BinlogScanNode binlogScanNode = new BinlogScanNode(context.getNextNodeId(), tupleDescriptor);
+            binlogScanNode.computeStatistics(optExpr.getStatistics());
+
+            // Add OPS column of binlog
+            SlotDescriptor opsSlot = context.getDescTbl().addSlotDescriptor(tupleDescriptor);
+            opsSlot.setIsNullable(false);
+            opsSlot.setType(Type.TINYINT);
+            opsSlot.setLabel("ops");
+            ColumnRefOperator columnRef =
+                    new ColumnRefOperator(opsSlot.getId().asInt(), opsSlot.getType(), "ops", opsSlot.getIsNullable());
+            context.getColRefToExpr().put(columnRef, new SlotRef(opsSlot.getLabel(), opsSlot));
```

Comment:
So here we still image there is a `ops` columns in the Binlog? How to handle the hidden columns?